source/macos/AdaptiveCards/AdaptiveCards/Views/ACRDateField.swift

```diff
@@ -60,6 +70,9 @@ class ACRDateField: NSView, InputHandlingViewProtocol {
             if let dateValue = dateValue, let date = dateFormatter.date(from: dateValue) {
                 textField.stringValue = dateFormatterOut.string(from: date)
                 selectedDate = date
+                clearButton.isHidden = false
+            } else {
+                clearButton.isHidden = true
             }
```

Comment:
```suggestion
            } else {
                textField.stringValue = ""
                clearButton.isHidden = true
            }
```

Reply:
The string value is already blank, that is why I didn't initialise it